mlflow/utils/databricks_utils.py

```diff
@@ -953,6 +952,9 @@ def _get_databricks_creds_config(tracking_uri):
                 config = _config
                 break
 
+    if ignore_error:
```

Comment:
what does this mean?

Reply:
it means if can't read valid credentials, does not raise error.

Reply:
can you insert a comment? `ignore_error` sounds ambigous.